codalab/rest/users.py

```diff
@@ -110,3 +117,22 @@ def fetch_users():
     usernames.discard('')  # str.split(',') will return '' on empty strings
     users = local.model.get_users(usernames=(usernames or None))
     return UserSchema(many=True).dump(users).data
+
+
+@patch('/users')
+def update_users():
+    """Update users."""
+    if request.user.user_id != local.model.root_user_id:
```

Comment:
I agree it can be confusing, I'm open to suggestions for better ideas.

I'm currently just following the ideas in GitHub's users API:
https://developer.github.com/v3/users/